### DISCUSSION TITLE
door lock schedules are not deleting sometimes

### DISCUSSION THREAD

**Discussion by priyanka043** [zwave-js/zwave-js-ui#3209] [Q&A]
Hi,

I have tried to add the zwave door lock weekday schedules, after addition
![Screenshot (33)](https://github.com/zwave-js/zwave-js-ui/assets/139616028/1b77b635-72ae-49e5-a722-e824c98477e5)
, i tried to delete that schedule but after some trials only it is getting deleted, but even though it is not removed, i can see that slot removed.

**Comment by robertsLando** [comment]
Wait for: https://github.com/zwave-js/zwave-js-ui/pull/3176
